worldguard-legacy/src/main/java/com/sk89q/worldguard/bukkit/WorldGuardPlugin.java

```diff
@@ -129,6 +132,19 @@ public void onEnable() {
         configureLogger();
 
         getDataFolder().mkdirs(); // Need to create the plugins/WorldGuard folder
+        
+        // Load in custom flags
+        for (Plugin plugin : this.getServer().getPluginManager().getPlugins()) {
+            CustomFlagBroker broker = new CustomFlagBrokerImpl();
```

Comment:
Oh I hadn't noticed it's done for every plugin. Yeah that is bad. I thought it was outside the loop...
But inside the if is probably better in conjunction with my suggestion in a comment a few lines below this one.